data_deletion/__init__.py

```diff
@@ -200,3 +223,32 @@ def mark_as_deleted(self):
 
     def __repr__(self):
         return self.sample_id + ' (%s)' % self.release_date
+
+
+class FinalSample(ProcessedSample):
+
+    @cached_property
+    def files_to_purge(self):
+        _file_to_purge = []
+        if self.released_data_folder:
+            _file_to_purge.extend(util.find_files(self.released_data_folder, '*'))
+        raw_files = self.raw_data_files
+        if raw_files:
+            _file_to_purge.extend(raw_files)
+
+        processed_files = self.processed_data_files
+        if processed_files:
+            _file_to_purge.extend(processed_files)
+
+        unreleased_files = [f for f in _file_to_purge if not is_released(f)]
+        if unreleased_files:
+            raise ArchivingError('Files not yet remove from lustre cannot be removed from tape: %s' % unreleased_files)
+
+        return _file_to_purge
+
+    @cached_property
+    def files_to_remove_from_lustre(self):
+        return []
```

Comment:
Do we need this?

Reply:
The parent class has a cached property that would return the bam and vcf file.
This is to ensure that no external call to this return an inadequate list of file.